OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -72,6 +72,9 @@ public class AircraftInfo : ITraitInfo, IPositionableInfo, IFacingInfo, IMoveInf
 		[Desc("Can the actor hover in place mid-air? If not, then the actor will have to remain in motion (circle around).")]
 		public readonly bool CanHover = false;
 
+		[Desc("Can the actor change direction without having to change facing? If not, the actor will have to fly a curve to change direction.")]
+		public readonly bool CanChangeDirectionWithoutTurning = false;
```

Comment:
This name is a bit long and unclear, IMO. How about inverting its meaning and calling it something like `TurnToFly` or `FlyForwards`?